src/Hopper/Internal/LoweredCore/EvalClosureConvertedANF.hs

```diff
@@ -531,6 +533,14 @@ enterTotalMathPrimopSimple controlstack (opId, refs) = do
         argAsLiteral (ValueLitCC lit) = Right lit
         argAsLiteral notLit = Left $ "Not a literal: `" ++ show notLit ++ "`"
 
+enterCaseCC
+  :: SymbolRegistryCC
+  -> EnvStackCC
+  -> ControlStackCC
+  -> (Variable, [(ConstrId, Word32, V.Vector BinderInfoCC, AnfCC)])
```

Comment:
i see, this lines up with `CaseCC var () branches -> enterCaseCC symbolReg envStk stack (var, branches)`